src/common/main.lua

```diff
@@ -24,6 +24,7 @@ function ant.init()
 	Initialized = Initialized or false
 	-- INSERT placeholder used by build script to inject the appropriate ID
 	SourceCodeTxId = SourceCodeTxId or "__INSERT_SOURCE_CODE_ID__"
+	AntRegistryId = AntRegistryId or ao.env.Process.Tags["ANT-Registry-Id"] or nil
```

Comment:
nit - extra nil not necessary
```suggestion
	AntRegistryId = AntRegistryId or ao.env.Process.Tags["ANT-Registry-Id"]
```